roxctl/common/environment/environment-impl.go

```diff
@@ -79,12 +79,12 @@ func (c *cliEnvironmentImpl) HTTPClient(timeout time.Duration, authMethod ...aut
 }
 
 // GRPCConnection returns the common.GetGRPCConnection
-func (c *cliEnvironmentImpl) GRPCConnection() (*grpc.ClientConn, error) {
+func (c *cliEnvironmentImpl) GRPCConnection(retryTimeout time.Duration) (*grpc.ClientConn, error) {
```

Comment:
I don't really like that this is now _required_ as a parameter.

Let's make this one optional, and by default assume `time.Duration(0)` so not _every_ command is _required_ to add the retry timeout flag.

For making it optional:
You can either use the variadic args (like we do within e.g. `HTTPClient()`) or introduce an option for that. I'd keep it simple and use variadic args, but feel free to choose differently.